tests/testthat/test-SQLServer.R

```diff
@@ -173,6 +173,26 @@ test_that("SQLServer", {
     expect_equal(as.double(values[[6]]), as.double(received[[6]]))
   })
 
+  local({
+    con <- DBItest:::connect(DBItest:::get_default_context())
+    input <- DBI::SQL(c(
+               "testtable",
+               "[testtable]",
+               "[testschema].[testtable]",
+               "[testschema].testtable",
+               "[testdb].[testschema].[testtable]",
+               "[testdb].[testschema].testtable" ))
```

Comment:
Maybe it's getting to exotic but do we need a test for a keyword containing a `"` or `[`?

Reply:
Thanks; added a few more.